skopt/space/space.py

```diff
@@ -46,6 +46,9 @@ def check_dimension(dimension, transform=None):
         the returned Dimension instance the values are scaled
         between 0 and 1.
 
+        If transform is set to "Identity", when `transform` is called
```

Comment:
also the accepted values for `transform` need to be updated in the docstring just above (line 44)